src/components/Home.js

```diff
@@ -1,8 +1,27 @@
 import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
+import React, { useEffect, useState } from 'react';
+import axiosInstance from '../apis/axiosClient';
 
 function Home() {
   const { status, user } = useAuth();
 
+  // testing axiosInstance request
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    axiosInstance
+      .get('/users/showMe')
+      .then((response) => {
+        setData(response.data); // Set the state with the response data
+        console.log('HTTP response data:', response.data);
+      })
+      .catch((error) => {
+        console.error('Error fetching data:', error);
+      });
+  }, []);
+
+  // end of testing
+
```

Comment:
Tested, it is working; feel free to remove this bit before merging :+1:

Reply:
I removed that